airbyte_cdk/sources/streams/http/requests_native_auth/abstract_oauth.py

```diff
@@ -396,6 +396,7 @@ def _log_response(self, response: requests.Response) -> None:
                     "Obtains access token",
                     self._NO_STREAM_NAME,
                     is_auxiliary=True,
+                    type="AUTH",
```

Comment:
❓ Given that we have separate types for the 3 `async` requests. Should we have separate types for the different oauth requests? get token vs refresh token?

Reply:
For me, it stands for the same purpose - Authentication == AUTH type of the request. It will be available in the `auxiliary_requests` (global ones/test read). I don't see the value in splitting into AUTH == Regular Authentication and OAUTH_AUTH == OAuth Authentication - feels completely the same to me. I'd leave a single type here.